redisinsight/ui/src/packages/redisgraph/src/Graph.tsx

```diff
@@ -1,14 +1,25 @@
-import React, { useEffect, useRef, useState, useMemo } from 'react';
+import { useEffect, useRef, useState, useMemo } from 'react';
```

Comment:
Also could you everywhere (timeseries, grapm) remove semicolon?
We don't use them in the Front-end side